graphql_jwt/refresh_token/mixins.py

```diff
@@ -26,12 +28,16 @@ def refresh(cls, root, info, refresh_token, **kwargs):
 
         payload = jwt_settings.JWT_PAYLOAD_HANDLER(refresh_token.user, context)
         token = jwt_settings.JWT_ENCODE_HANDLER(payload, context)
-
         refresh_token.rotate(context)
-        refreshed_token = refresh_token_lazy(refresh_token.user)
 
         if getattr(context, 'jwt_cookie', False):
-            info.context.jwt_refresh_token = refreshed_token
+            context.jwt_refresh_token = create_refresh_token(
+                refresh_token.user,
+                context,
```

Comment:
@mongkok should the `context` variable be here? 

Until I have commented line 36 I got an error:
```
graphql.error.located_error.GraphQLLocatedError: create_refresh_token() takes 1 positional argument but 2 were given
````

Reply:
Thanks @hansek.